src/DynamicRupture/Typedefs.hpp

```diff
@@ -40,6 +51,7 @@ struct FaultStresses {
   alignas(ALIGNMENT) real normalStress[CONVERGENCE_ORDER][misc::numPaddedPoints] = {{}};
   alignas(ALIGNMENT) real traction1[CONVERGENCE_ORDER][misc::numPaddedPoints] = {{}};
   alignas(ALIGNMENT) real traction2[CONVERGENCE_ORDER][misc::numPaddedPoints] = {{}};
+  alignas(ALIGNMENT) real fluidPressure[CONVERGENCE_ORDER][misc::numPaddedPoints] = {{}};
```

Comment:
Maybe `#ifdef USE_POROELASTIC`? Though I'm not sure if it's really necessary here...

Reply:
Same thing with ifdefs, would like to reduce their usage as much as possible. One ifdef in one place, needs five more ifdefs in other places, to make compilation always possible.